pkg/apis/postgres-operator.crunchydata.com/v1/pgbackrest_types.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 - 2025 Crunchy Data Solutions, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+)
+
+// PGBackRestArchive defines a pgBackRest archive configuration
```

Comment:
Do we also want to add some CEL validation in V1 to disallow setting log path through the `global` map?

Reply:
I guess we could... Obviously can't stop users from setting log path via the `configuration`...

Reply:
Latest commit has CEL validation to disallow setting log-path via global.